docs/src/apalache/features.md

```diff
@@ -85,10 +85,10 @@ Note that our type system distinguishes tuples from general functions.
 
 Operator  | Supported? | Milestone | Comment
 ------------------------|:------------------:|:---------------:|--------------
-`e[i]` | ✔ / ✖ | - | Provided that ``i`` is a constant expression
-`<< e1, ..., e_n >>` | ✔ | - | By default, a tuple is constructed. Use a [type annotation](types-and-annotations.md) to construct a sequence of proper type.
+`e[i]` | ✔ / ✖ | - | Provided that `i` is a constant expression
+`<< e1, ..., e_n >>` | ✔ | - | By default, a tuple is constructed. Use a [type annotation](types-and-annotations.md) to construct a sequence.
```

Comment:
This is not even true anymore. You would need a type annotation in the both cases.

Reply:
I'll fix this. Thanks.